NEWS.md

```diff
@@ -1,3 +1,16 @@
+# devel
+
+## New features
+
+* The `rhub_check_list` class gained a `get_group()` and a `get_check()` methods returning a `rhub_check` object from the list, by group_id or check_id.
+
+* The `rhub_check` class gained a `browse()` method replacing the `web()` method for opening the URLs corresponding to a `rhub_check` object.
+
+* In `list_package_checks()` and `list_my_checks()` the `howmany` arguments now refers to checks submitted simultaneously, so if you submitted via `check_for_cran()` although it'd have created 3 builds, it'd count for one check as far as `howmany` is concerned, due to grouping in the API.
```

Comment:
I think we can leave this out, not a big deal.

Reply:
I removed this paragraph, thanks.